easybuild/tools/github.py

```diff
@@ -971,21 +973,24 @@ def push_branch_to_github(git_repo, target_account, target_repo, branch):
     if dry_run:
         print_msg(push_branch_msg + ' [DRY RUN]', log=_log)
     else:
-        print_msg(push_branch_msg, log=_log)
-        try:
-            res = remote.push(branch)
-        except GitCommandError as err:
-            raise EasyBuildError("Failed to push branch '%s' to GitHub (%s): %s", branch, github_url, err)
+        if target_account is None:
+            raise EasyBuildError("No valid GitHub username (--github-user) given, pushing branch will fail!")
```

Comment:
`target_account` could also be set some other way, there's no direct relation to `--github-user` here, so this error shouldn't be raised here...

Reply:
see https://github.com/ocaisa/easybuild-framework/pull/46

Reply:
Merged